microsoft-365/security/defender/advanced-hunting-query-results.md

```diff
@@ -69,6 +69,13 @@ AlertInfo
 
 When rendering the results, a column chart displays each severity value as a separate column:
 
+
+```kusto
+AlertInfo
+| summarize Total = count() by Severity
+| render coloumnchart
```

Comment:
col**o**umnchart > columnchart

Once this is changed, it should be good to go.